admin_guide/scheduling/custom_scheduler.adoc

```diff
@@ -16,26 +16,99 @@ You can run multiple, custom schedulers alongside the default scheduler and conf
 
 To schedule a given pod using a specific scheduler, xref:admin-guide-sched-custom-deploy[specify the name of the scheduler in that pod specification].
 
-[[admin-guide-sched-custom-deploy]]
-== Deploying the Scheduler
-
-The steps below are the general process for deploying a scheduler into your cluster.
-
 [NOTE]
 ====
-Information on how to create/deploy a scheduler is outside the scope of this document. For an example, see link:https://github.com/kubernetes/kubernetes/tree/master/pkg/scheduler[*_plugin/pkg/scheduler_*] in the Kubernetes source directory.
+Information on how to create the scheduler binary is outside the scope of this document. For an example, see link:https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers[Configure Multiple Schedulers] in the Kubernetes documentation.
 ====
 
+[[admin-guide-sched-custom-package]]
+== Package the scheduler
+
+The general process for including a custom scheduler in your cluster
+involves creating an image and including that image in a deployment.
+
+. Package your scheduler binary into a container image. 
+
+. xref:../../creating_images/guidelines.html#creating-images-guidelines[Create a container image] containing the scheduler binary. 
++
+For example:
++
+----
+FROM <source>
+ADD <path-to-binary> /usr/local/bin/kube-scheduler
+----
+
+. Save the file as Dockerfile, build the image, and push it to a registry. 
++
+For example:
++
+----
+docker build -t <dest_env_registry_ip>:<port>/<namespace>/<image name>:<tag>
+docker push <dest_env_registry_ip>:<port>/<namespace>/<image name>:<tag>
+----
+
+. In {product-title}, create a deployment for the custom scheduler.
++
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: custom-scheduler
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: custom-scheduler
+subjects:
+- kind: ServiceAccount
+  name: custom-scheduler
+  namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: system:kube-scheduler
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: custom-scheduler
+  namespace: kube-system
+  labels:
+    app: custom-scheduler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: custom-scheduler
+  template:
+    metadata:
+      labels:
+        app: custom-scheduler
+    spec:
+      serviceAccount: custom-scheduler
+      containers:
+        - name: custom-scheduler
+          image: "<namespace>/<image name>:<tag>" <1>
+          imagePullPolicy: Always
+----
+<1> Specify the container image you created for the custom scheduler.
+
+[[admin-guide-sched-custom-deploy]]
+== Deploying Pods using a Custom Scheduler
```

Comment:
Minor: Pod ---> pod?

It looks like capitalization for the word "pod" is a bit inconsistent throughout.